src/components/FormField/FormField.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+import { FormFieldStyled, ErrorSpan, Label, TextArea } from "components/FormField/FromField.styled";
+
+export interface FormFieldProps {
+  field: any;
+  form: {touched: any, errors: any};
+  type: string;
+  placeholder: string;
+  step?: number
+}
+
+const FormField: React.FC<FormFieldProps> = (props) => {
+  return (
+    <>
+      <Label htmlFor={props.field.name}>{props.field.name}</Label>
+      {props.type === 'textarea' ?
```

Comment:
There is no need for it to be all-in-one. It's much more flexible to have components for inputs, textarea, select, etc and then use them in place, even at the cost of repeating yourself with passing same props, same validation messages, etc.